### ISSUE TITLE
2.11.0 stable release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#3622]
- #3499 
- #3446 
- #3587 
- #3589 
- #3592 
- #3577
- #3477 
- #3600 
- #3603
- #3598 

## Pick list
```
git cherry-pick 353b8cd5d9233a891637667f99a61fecbc7add92
git cherry-pick 6663cf5c02017244028625506bb898c32819af52
git cherry-pick 4a63ceedd3346bb58361437eceed529ca4d5abf1
git cherry-pick aec34ffd9fe24c938593454acdd58105a348e3cb
git cherry-pick 86c71872db08a9d837c7c5f45d1efbfc8432ea3f
git cherry-pick d0c178983aef39db41c3636bdf951fe80d8bd2a2
git cherry-pick e349b0e24db8325aa7fd5301a5159b53f20b992a
git cherry-pick 37ef33f643ad5db12ac00260e3d445258373a446
git cherry-pick fb03b13e2142c3420c9661ed16b72b4813b7e9cb
git cherry-pick 9a2ac0ca99286e6a67e163d6ab9b15a5d36bacfd
```

**Comment by joeparsons** [comment]
## Release notes snippet
`git log 2.11.0-beta1..HEAD --pretty='format:- %s'`
```
## Changes since 2.11.0-beta1

- Closes #3597: Add lifecycle_link to experimental modules documentation in experimental modules. (#3598)
- Close #3602 transpile JS with build tools `lando yarn run build` (#3603)
- Fix #3599 Can no longer save text format and editor configuration form, and upgrade bootstrap_utilities from 2.0.0 to 3.0.0 (#3600)
- Closes #3473: Adding policy recommendations for Experimental Modules (#3477)
- Closes #3567 Styling adjustments for the AZ Finder exposed form (AZDIGITAL-158) (#3577)
- Fixes #3591 Add NetID Unique Constraint (#3592)
- Close #3588 Allow newly enabled module config to be overridden. (#3589)
- Close #3586 Set lifecycle key in info files of experimental modules. (#3587)
- Closes #3445 Make config dependencies, module dependencies in az_paragraphs:az_paragraphs_view (#3446)
- Closes #3498 Make config dependencies, module dependencies in az_seo. (#3499)
- Back to dev.
```

**Comment by bberndt-uaz** [comment]
New Changes section:
```
## New changes (not included in 2.9.x or 2.10.x)

### Drupal Core Updates
- #3290: Upgrade Drupal core to 10.3.0 (#3416)
- Update drupal/core-recommended requirement from 10.3.0 to 10.3.1 (#3539)

### Bug Fixes
- #2778: Fix 10.3.x specific PHPCS issue in AZEnterpriseAttributesMigrationSync (#3515)
- Closes #3546 PHPUnit tests cannot be run locally using lando (#3547)
- Fixes #3556, Fixes #3557 Regressions in AZEventTrellisViewsField, AZTrellisEventSource (#3558)
- Fix #3599 Can no longer save text format and editor configuration form, and upgrade bootstrap_utilities from 2.0.0 to 3.0.0 (#3600)
- Close #3602 transpile JS with build tools `lando yarn run build` (#3603)

### General Enhancements
- Closes #3096 Simple file system configuration change to sanitize uploaded filenames (#3180)
- Closes #3262 Add ability for content admins to set finder overrides (#3420)
- #2635, #3290: Add WebP conversion step to all included image styles (#3418)
- Closes #3396: Makes clickable cards more accessible (#3463)
- Closes #3335: Remove modules deprecated in 2.9.x (#3510)
- Close #3481 Remove az_google_tag's dependency on ctools and remove ctools from az_quickstart (#3481)
- Close #3329 Update prepare repository for minor release action so that it updates the main branch version alias as a pull request (#3337)
- Closes #3174 Made text area not required on az_text_background (#3392)
- Closes #3365 Set Easy Breadcrumb preferred menu setting to main menu by default (#3366)
- Close #3385 Remove redundant permissions in "Content Administrator" role (#3388)
- Update Arizona Icons to 1.0.2 (#3414)
- #3290: Convert plugin annotations to attributes for plugins updated in 10.3.0 (#3419)
- Closes #3561: Add NetID field to AZ Person. (#3562)
- Fixes #3591 Add NetID Unique Constraint (#3592)

### Integrations
- Fixes #2778 Cron Migration settings for Enterprise Attributes (#2781)
- Fixes #3305 Provide Attribute Filtering for News Export (#3468)
- Fixes #3470 Additional Course Fields (#3495)

### Contrib Module Updates
- Close #3467 Upgrade Blazy/Slick to 3.x (#3467)
- Closes #2835: Switch back to d.o. version of smart_title. (#3514)
- Update citation-style-language/styles requirement from 0.0.411 to 0.0.413 (#3402)
- Update citation-style-language/styles requirement from 0.0.413 to 0.0.434 (#3459)
- Update drupal/access_unpublished requirement from 1.5.0 to 1.6.0 (#3533)
- Update drupal/auto_entitylabel requirement from 3.0.0 to 3.1.0 (#3469)
- Update drupal/auto_entitylabel requirement from 3.0.0 to 3.2.0 (#3478)
- Update drupal/better_exposed_filters requirement from 6.0.5 to 6.0.6 (#3456)
- Update drupal/blazy requirement from 3.0.8 to 3.0.9 (#3513)
- Update drupal/cas requirement from 2.2.0 to 2.3.2 (#3417)
- Update drupal/chosen requirement from 4.0.0 to 4.0.1 (#3431)
- Update drupal/ckeditor_bs_grid requirement from 2.0.11 to 2.0.12 (#3538)
- Update drupal/config_readonly requirement from 1.0.0-beta5 to 1.0.0 (#3536)
- Update drupal/crop requirement from 2.3.0 to 2.4.0 (#3540)
- Update drupal/easy_breadcrumb requirement from 2.0.6 to 2.0.7 (#3509)
- Update drupal/externalauth requirement from 2.0.5 to 2.0.6 (#3543)
- Update drupal/google_tag requirement from 2.0.4 to 2.0.5 (#3437)
- Update drupal/jquery_ui requirement from 1.6.0 to 1.7.0 (#3458)
- Update drupal/jquery_ui_datepicker requirement from 2.0.0 to 2.1.0 (#3474)
- Update drupal/media_library_form_element requirement from 2.0.6 to 2.1.0 (#3534)
- Update drupal/menu_block requirement from 1.10.0 to 1.11.0 (#3407)
- Update drupal/migrate_plus requirement from 6.0.2 to 6.0.4 (#3555)
- Update drupal/paragraphs_admin requirement from 1.5.0 to 1.6.0 (#3401)
- Update drupal/smart_date requirement from 4.0.3 to 4.1.3 (#3386)
- Update drupal/smtp requirement from 1.2.0 to 1.3.0 (#3508)
- Update drupal/views_remote_data requirement from 1.0.2 to 1.0.3 (#3506)
- Update drupal/viewsreference requirement from 2.0.0-beta7 to 2.0.0-beta8 (#3460)
- Update drupal/webform requirement from 6.2.3 to 6.2.7 (#3559)

### Dependency Updates
- Update composer/installers requirement from 2.2.0 to 2.3.0 (az-quickstart-scaffolding)
- Closes #3151 Update NodeJS in ProboCI config to version 20.13.0 (#3152)
```

**Comment by bberndt-uaz** [comment]
Draft of release notes:

```
## Important Notes

### 1. End of Security Support for Quickstart 2.9.x
_**Quickstart 2.9.x will no longer receive security support. Sites on Quickstart 2.9.x or earlier should upgrade to a supported release as soon as possible.**_ See our [release policy](https://github.com/az-digital/az_quickstart/blob/main/RELEASES.md#release-overview) for more information.

### 2. Update to Drupal 10.3
Quickstart 2.11.0 includes an update from Drupal Core 10.2 to Drupal Core 10.3.1. Be sure to read both the [10.3.0 release notes](https://www.drupal.org/project/drupal/releases/10.3.0) and the [10.3.1 release notes](https://www.drupal.org/project/drupal/releases/10.3.1).

### 3. Removal of Deprecated Modules
Modules deprecated as of 2.9.x have been removed from Quickstart 2.11.x and should be uninstalled on sites before upgrading to 2.11.x (unless they have been added to a site-specific composer.json file).
  - `block_content_permissions`

### 4. Chaos Tools (`ctools`) Dependency Removed from Quickstart 2

Quickstart no longer has a runtime dependency on the ctools module, and the module has been slated for removal. This might cause problems if other modules implicitly use ctools functionality without depending on it.

Note that for backwards compatibility, the composer definition still depends on the ctools project, but it's no longer installed on new projects and existing projects may uninstall it if they are not using it.

The composer definition is there so that existing sites do not automatically have the code removed underneath them and continue to work. This will be removed in the future in a new major version. It is recommended to either add an explicit dependency on ctools if you plan to keep using it or uninstall it.

To fully remove ctools from the project, add a replace section to the root composer.json:

```json
"replace": {
  "drupal/ctools": "*"
}
```

Note that this will also skip it in case another module depends on it and the definition will need to be removed then.

Before uninstalling ctools, verify that no configuration dependencies remain on views or other config entities. Views should have that dependency removed automatically when resaving them.

### 5. WebP Image Conversion
This release adds a WebP conversion step to all image styles provided by AZ Quickstart and [Drupal core](https://www.drupal.org/node/3421405). A [patch for Drupal core](https://www.drupal.org/project/drupal/issues/3310963) has also been included that adds a directive to the `.htaccess` file included with Drupal core to ensure the WebP image MIME type is supported. Sites hosted on Apache web servers may need to [update their .htaccess files](https://www.drupal.org/node/3456338) to take advantage of this improvement.

### 6. New NetID field on AZ Person
A new _NetID_ field has been added to the AZ Person content type. Sites that have added additional fields to the AZ Person content type may need to adjust the field display and form display settings for the AZ Person content type after applying configuration changes from the distribution.  This new field will be needed for planned future AZQS integrations and enhancements.


## How to Update
- **Important**: [Database updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-database-updates) and [Distribution configuration updates](https://github.com/az-digital/az_quickstart/wiki/Maintaining-Quickstart-2-Sites#applying-distribution-configuration-updates) should **always** be applied when updating Quickstart sites.
- **Important**: Updating from a Quickstart codebase version less than 2.6.x is no longer supported. To update to a Quickstart 2.11.x release (e.g. 2.11.0) from a version less than 2.6.8 you must first update to 2.6.8 or greater and run database updates. Database updates added to Quickstart prior to version 2.6.0 have been removed in 2.7.x, 2.8.x, and 2.9.x. Also, some core modules included in Drupal 9 have been removed in Drupal 10. For more relevant info about upgrading from 2.6.x, see [Quickstart 2.7.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.7.0-Upgrade-Notes).
- **For Site Updaters Managing Their Own Environments**: With the inclusion of a minor Drupal update in this release, it's imperative for site updaters who manage their own environments to thoroughly review the Drupal release notes and their own setups. Please review https://github.com/az-digital/az-quickstart-scaffolding for the latest recommendations.
- Refer to [Quickstart 2.11.0 Upgrade Notes](https://github.com/az-digital/az_quickstart/wiki/Quickstart-2.11.0-Upgrade-Notes) on the wiki for more information about updating to 2.11.


## Changes since 2.11.0-beta1
- Closes #3597: Add lifecycle_link to experimental modules documentation in experimental modules. (#3598)
- Close #3602 transpile JS with build tools `lando yarn run build` (#3603)
- Fix #3599 Can no longer save text format and editor configuration form, and upgrade bootstrap_utilities from 2.0.0 to 3.0.0 (#3600)
- Closes #3473: Adding policy recommendations for Experimental Modules (#3477)
- Closes #3567 Styling adjustments for the AZ Finder exposed form (AZDIGITAL-158) (#3577)
- Fixes #3591 Add NetID Unique Constraint (#3592)
- Close #3588 Allow newly enabled module config to be overridden. (#3589)
- Close #3586 Set lifecycle key in info files of experimental modules. (#3587)
- Closes #3445 Make config dependencies, module dependencies in az_paragraphs:az_paragraphs_view (#3446)
- Closes #3498 Make config dependencies, module dependencies in az_seo. (#3499)


## New Changes in 2.11.0 (not included in 2.9.x or 2.10.x)

### Drupal Core Updates
- #3290: Upgrade Drupal core to 10.3.0 (#3416)
- Update drupal/core-recommended requirement from 10.3.0 to 10.3.1 (#3539)

### Bug Fixes
- #2778: Fix 10.3.x specific PHPCS issue in AZEnterpriseAttributesMigrationSync (#3515)
- Closes #3546 PHPUnit tests cannot be run locally using lando (#3547)
- Fixes #3556, Fixes #3557 Regressions in AZEventTrellisViewsField, AZTrellisEventSource (#3558)
- Fix #3599 Can no longer save text format and editor configuration form, and upgrade bootstrap_utilities from 2.0.0 to 3.0.0 (#3600)
- Close #3602 transpile JS with build tools `lando yarn run build` (#3603)

### General Enhancements
- Closes #3096 Simple file system configuration change to sanitize uploaded filenames (#3180)
- Closes #3262 Add ability for content admins to set finder overrides (#3420)
- #2635, #3290: Add WebP conversion step to all included image styles (#3418)
- Closes #3396: Makes clickable cards more accessible (#3463)
- Closes #3335: Remove modules deprecated in 2.9.x (#3510)
- Close #3481 Remove az_google_tag's dependency on ctools and remove ctools from az_quickstart (#3481)
- Close #3329 Update prepare repository for minor release action so that it updates the main branch version alias as a pull request (#3337)
- Closes #3174 Made text area not required on az_text_background (#3392)
- Closes #3365 Set Easy Breadcrumb preferred menu setting to main menu by default (#3366)
- Close #3385 Remove redundant permissions in "Content Administrator" role (#3388)
- Update Arizona Icons to 1.0.2 (#3414)
- #3290: Convert plugin annotations to attributes for plugins updated in 10.3.0 (#3419)
- Closes #3561: Add NetID field to AZ Person. (#3562)
- Fixes #3591 Add NetID Unique Constraint (#3592)

### Integrations
- Fixes #2778 Cron Migration settings for Enterprise Attributes (#2781)
- Fixes #3305 Provide Attribute Filtering for News Export (#3468)
- Fixes #3470 Additional Course Fields (#3495)

### Contrib Module Updates
- Close #3467 Upgrade Blazy/Slick to 3.x (#3467)
- Closes #2835: Switch back to d.o. version of smart_title. (#3514)
- Update citation-style-language/styles requirement from 0.0.411 to 0.0.413 (#3402)
- Update citation-style-language/styles requirement from 0.0.413 to 0.0.434 (#3459)
- Update drupal/access_unpublished requirement from 1.5.0 to 1.6.0 (#3533)
- Update drupal/auto_entitylabel requirement from 3.0.0 to 3.1.0 (#3469)
- Update drupal/auto_entitylabel requirement from 3.0.0 to 3.2.0 (#3478)
- Update drupal/better_exposed_filters requirement from 6.0.5 to 6.0.6 (#3456)
- Update drupal/blazy requirement from 3.0.8 to 3.0.9 (#3513)
- Update drupal/cas requirement from 2.2.0 to 2.3.2 (#3417)
- Update drupal/chosen requirement from 4.0.0 to 4.0.1 (#3431)
- Update drupal/ckeditor_bs_grid requirement from 2.0.11 to 2.0.12 (#3538)
- Update drupal/config_readonly requirement from 1.0.0-beta5 to 1.0.0 (#3536)
- Update drupal/crop requirement from 2.3.0 to 2.4.0 (#3540)
- Update drupal/easy_breadcrumb requirement from 2.0.6 to 2.0.7 (#3509)
- Update drupal/externalauth requirement from 2.0.5 to 2.0.6 (#3543)
- Update drupal/google_tag requirement from 2.0.4 to 2.0.5 (#3437)
- Update drupal/jquery_ui requirement from 1.6.0 to 1.7.0 (#3458)
- Update drupal/jquery_ui_datepicker requirement from 2.0.0 to 2.1.0 (#3474)
- Update drupal/media_library_form_element requirement from 2.0.6 to 2.1.0 (#3534)
- Update drupal/menu_block requirement from 1.10.0 to 1.11.0 (#3407)
- Update drupal/migrate_plus requirement from 6.0.2 to 6.0.4 (#3555)
- Update drupal/paragraphs_admin requirement from 1.5.0 to 1.6.0 (#3401)
- Update drupal/smart_date requirement from 4.0.3 to 4.1.3 (#3386)
- Update drupal/smtp requirement from 1.2.0 to 1.3.0 (#3508)
- Update drupal/views_remote_data requirement from 1.0.2 to 1.0.3 (#3506)
- Update drupal/viewsreference requirement from 2.0.0-beta7 to 2.0.0-beta8 (#3460)
- Update drupal/webform requirement from 6.2.3 to 6.2.7 (#3559)

### Dependency Updates
- Update composer/installers requirement from 2.2.0 to 2.3.0 (az-quickstart-scaffolding)
- Closes #3151 Update NodeJS in ProboCI config to version 20.13.0 (#3152)
```

**Comment by joeparsons** [comment]
Release created:
https://github.com/az-digital/az_quickstart/releases/tag/2.11.0
